kibana/lib/openshift_logging_plugin/public/hack.js

```diff
@@ -0,0 +1,67 @@
+import chrome from 'ui/chrome';
+import { uiModules } from 'ui/modules';
+const app = uiModules.get('apps/openshift_logging_plugin/index');
+
+function redirectOnSessionTimeout($window) {
+    document.cookie = "name=_oauth_proxy; expires=Thu, 01 Jan 1970 00:00:00 GMT; path=/;"
```

Comment:
Why does this function need to override the cookies here?

Reply:
In testing, the experience I saw when the oauth token was set really log (i.e. 1m) and left the cookie as is (24h) the user would be redirected as desired.  If i tried to refresh, however, screen goes blank with a 401.  This was only resolvable by removing the cookie.  Here I'm attempting to expire the cookie to avoid that experience though I'm still testing to confirm it is satisfactory.

Reply:
Discussion with the console this does nothing because its an http cookie.  removing